### PR TITLE
Convert the hand tool, interfaces and secondary toolbar to ES6 syntax

### PR DESCRIPTION
**Pull request by timvandermeij**
*Easier reviewing with https://github.com/mozilla/pdf.js/pull/8391/files?w=1.*

### CODE REVIEW

File: web/hand_tool.js [hunk 1]
```diff
@@ -46,12 +43,12 @@ var HandTool = (function HandToolClosure() {
 
     this.eventBus.on('togglehandtool', this.toggle.bind(this));
 
-    Promise.all([localized,
-                 preferences.get('enableHandToolOnLoad')]).then((values) => {
+    let enableOnLoad = preferences.get('enableHandToolOnLoad');
+    Promise.all([localized, enableOnLoad]).then((values) => {
```
Comment (Snuffleupagus) [inline web/hand_tool.js]: Note that you can actually use array destructuring here (and I'm not sure that the temporary `enableOnLoad` is necessary for readability); how about the following instead?
```javascript
Promise.all([
  localized,
  preferences.get('enableHandToolOnLoad')
]).then(([, value]) => {
  if (value === true) {
    this.handTool.activate();
  }
}).catch(function(reason) {});
```

**Review by Snuffleupagus — APPROVED** [review]
I've got one small suggestion, but apart from that this looks really good; thank you!

File: web/hand_tool.js (hunk 1 above)
Comment (yurydelendik) [inline web/hand_tool.js]: I had trouble with `([, value]) =>` with construct. Let's keep it as is -- I have something in mind to remove this statement later.

**Pull request merged**